<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeControlWidget</class>
 <widget class="QWidget" name="TimeControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="m_groupBox">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Mode</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="m_tournamentRadio">
        <property name="toolTip">
         <string>Tournament time control</string>
        </property>
        <property name="text">
         <string>To&amp;urnament</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_timePerMoveRadio">
        <property name="toolTip">
         <string>Fixed time limit for each move</string>
        </property>
        <property name="text">
         <string>Time &amp;per move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_infiniteRadio">
        <property name="toolTip">
         <string>Infinite thinking time</string>
        </property>
        <property name="text">
         <string>Infinite</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_hourglassRadio">
        <property name="toolTip">
         <string>Time used is added to opponent's available time</string>
        </property>
        <property name="text">
         <string>Hourglass</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Mo&amp;ves:</string>
       </property>
       <property name="buddy">
        <cstring>m_movesSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="m_movesSpin">
       <property name="toolTip">
        <string>The number of moves in a time control</string>
       </property>
       <property name="specialValueText">
        <string>Whole game</string>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Time:</string>
       </property>
       <property name="buddy">
        <cstring>m_timeSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QDoubleSpinBox" name="m_timeSpin">
         <property name="toolTip">
          <string>Time limit for the chosen time control</string>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="value">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="m_timeUnitCombo">
         <property name="toolTip">
          <string>Time unit for the time limit</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Seconds</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hours</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>In&amp;crement:</string>
       </property>
       <property name="buddy">
        <cstring>m_incrementSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="m_incrementSpin">
       <property name="toolTip">
        <string>Time increment per move in seconds</string>
       </property>
       <property name="suffix">
        <string> sec</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="m_pliesSpin">
       <property name="toolTip">
        <string>Maximum search depth in plies (engines only)</string>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>P&amp;lies:</string>
       </property>
       <property name="buddy">
        <cstring>m_pliesSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="m_nodesSpin">
       <property name="toolTip">
        <string>Maximum number of nodes to search (engines only)</string>
       </property>
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Nodes:</string>
       </property>
       <property name="buddy">
        <cstring>m_nodesSpin</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="m_marginSpin">
       <property name="toolTip">
        <string>The time limit can be exceeded by this many milliseconds</string>
       </property>
       <property name="suffix">
        <string> ms</string>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Mar&amp;gin:</string>
       </property>
       <property name="buddy">
        <cstring>m_marginSpin</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
